pkg/services/iothub/type.go

```diff
@@ -0,0 +1,26 @@
+package iothub
+
+import "github.com/Azure/open-service-broker-azure/pkg/service"
+
+type instanceDetails struct {
+	ARMDeploymentName string               `json:"armDeployment"`
+	IoTHubName        string               `json:"iotHubName"`
+	KeyName           string               `json:"keyName"`
+	Key               service.SecureString `json:"key"`
```

Comment:
Should we move the key info to binding details so that it wouldn't lead breaking change when adding underlying support for role bindings?

Reply:
You got me. For this PR, we can keep it simple and reserve the possibility.

Reply:
If so, if one day the user requires this feature, we can store all returned keys in `instanceDetails`, add a `BindingParametersSchema`, and extract user-specified key in binding. If we make the binding parameter is optional and default to `owner`, then all changes are invisible to users and no breaking change will happen.

Reply:
Sounds good.